src/ucp/core/ucp_ep.c

```diff
@@ -2528,6 +2528,32 @@ void ucp_ep_config_cleanup(ucp_worker_h worker, ucp_ep_config_t *config)
     ucs_free(config->key.dst_md_cmpts);
 }
 
+void ucp_ep_config_short_init(ucp_worker_h worker, ucp_ep_config_t *ep_config,
```

Comment:
minor: it is used in ucp_worker.c only, why not move it into that file?

Reply:
ok